snippets/csharp/System/NullReferenceException/Overview/Array2.cs

```diff
@@ -1,17 +1,17 @@
 ﻿// <Snippet9>
 using System;
 
-public class Example
+public class Array2Example
 {
-   public static void Main()
-   {
-      String[] values = { "one", null, "two" };
-      for (int ctr = 0; ctr <= values.GetUpperBound(0); ctr++)
-         Console.Write("{0}{1}",
-                       values[ctr] != null ? values[ctr].Trim() : "",
-                       ctr == values.GetUpperBound(0) ? "" : ", ");
-      Console.WriteLine();
-   }
+    public static void Main()
+    {
+        string[] values = { "one", null, "two" };
+        for (int ctr = 0; ctr <= values.GetUpperBound(0); ctr++)
+            Console.Write("{0}{1}",
+                          values[ctr] != null ? values[ctr].Trim() : "",
+                          ctr == values.GetUpperBound(0) ? "" : ", ");
```

Comment:
more string interpolation:

```suggestion
            Console.Write($"{(values[ctr] != null ? values[ctr].Trim() : ""}{(ctr == values.GetUpperBound(0) ? "" : ", "))}");
```